Tests/PostgresNIOTests/New/Messages/BackendKeyDataTests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 12.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```